src/components/Assistant.vue

```diff
@@ -213,7 +214,7 @@ export default {
 	},
 	computed: {
 		showAssistant() {
-			return !this.$isRichWorkspace && !this.$isPublic && window?.OCA?.TPAssistant?.openAssistantForm
+			return !this.$isRichWorkspace && !this.$isPublic && window?.OCA?.TpAssistant?.openAssistantForm
```

Comment:
@julien-nc Was there a casing change in the API here? Is it bound to a Nextcloud release?

Reply:
Sorry for the late answer. This was and will stay OCA.TPAssistant.